offerzen/global/company-fast-track-access/company-prospect-form.js

```diff
@@ -112,6 +112,15 @@
                 errorText.show();
                 clearTimeout(buttonLabelTimer);
               },
+              // 400 is returned if user company exists on OfferZen and is new user
```

Comment:
It's probably a good idea to add this to the staging scripts as well `offerzen/global/company-fast-track-access/company-prospect-form-staging.js`